src/effects/bounce.ts

```diff
@@ -9,22 +9,36 @@ export const bounce = buildEffect({
   requiresAnimation: true,
   params: [
     intParam({
-      name: 'Bounce Height',
-      description: 'Positive number',
-      defaultValue: (image) =>
-        image ? Math.floor(image.dimensions[1] / 10) : 10,
+      name: 'Bounce Height %',
+      description: 'Percentage of the image height',
+      defaultValue: 50,
       min: 0,
+      max: 100,
     }),
     bezierParam({
       name: 'Easing',
       defaultValue: miscUtil.LINEAR_BEZIER,
     }),
+    checkboxParam({
+      name: 'Up and Down',
+      description: 'If checked, the image will bounce in both directions',
+      defaultValue: false,
+    }),
   ] as const,
   fn: imageUtil.mapImageWithPrecompute(
-    ({ animationProgress, parameters: [height, easing] }) => {
+    ({
+      animationProgress,
+      parameters: [perc, easing, upAndDown],
+      dimensions: [_, height],
+    }) => {
       const b = miscUtil.bezierCurve(easing, true)(animationProgress);
+      const baseOffset = (perc / 100) * height;
       return {
-        yOffset: Math.round(height * b),
+        yOffset: Math.round(
+          upAndDown
```

Comment:
Hmm if we have upAndDown set, we then ignore the easing param. I would expect that doing upAndDown just means that the yOffset goes from `-baseOffset` to `+baseOffset` multipled by `b`, instead of `0` to `baseOffset * b`. The animation will need to be twice as fast then, so it still returns to the original position in the end.

The easing parameter should still let you get a more natural bouncing, no need to use `sin()`. One thing in my TODO is to make alternatives to `LINEAR_BEZIER` so we can default to something non-linear.